recipes/hera/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+  home: https://github.com/bioturing/hera
+  license: MIT
+  summary: hera is a bioinformatics tool that helps analyze RNA-seq data, providing base-to-base alignment BAM files, transcript abundance estimation, and fusion gene detection.
+
+package:
+  name: hera
+  version: 1.0
+
+requirements:
+  run:
+    - libgcc # [linux]
+    - python >=3
+
+test:
+  commands: (hera | grep VERSION 2>&1) > /dev/null
+
+source:
+  fn: hera-v1.0.linux.tar.gz #[linux]
+  url: https://github.com/bioturing/hera/releases/download/v1.0/hera-v1.0.linux.tar.gz #[linux]
```

Comment:
Yes, that's what I started with initially before resorting to the binaries. The build script downloads and statically compiles a few libraries (e.g. htslib, libhdf5); needs some patching to use the bioconda versions. I'll have a look at it tomorrow.